cli/tsc.rs

```diff
@@ -1000,20 +982,18 @@ pub async fn bundle(
   Ok(output)
 }
 
-/// This function is used by `Deno.compile()` and `Deno.bundle()` APIs.
-pub async fn runtime_compile<S: BuildHasher>(
+async fn create_runtime_module_graph(
   global_state: GlobalState,
   permissions: Permissions,
   root_name: &str,
-  sources: &Option<HashMap<String, String, S>>,
-  bundle: bool,
+  sources: &Option<HashMap<String, String>>,
   maybe_options: &Option<String>,
-) -> Result<Value, OpError> {
+) -> Result<(Vec<String>, HashMap<String, ModuleGraphFile>), OpError> {
```

Comment:
This return type is relatively difficult to understand. What's the first item (presumably module specifiers)? What's the key of the second item (also module specifiers...?)

Suggest defining a struct with some triple slash comments describing each component.

Reply:
Actually I already started cleaning this up in #6335

Reply:
I'll merge those changes into this PR